<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkLogHistory">
     <property name="text">
      <string>Enable Log History</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkChecksum">
     <property name="text">
      <string>Verify Checksum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCloseOnFinish">
     <property name="text">
      <string>Close window when finished</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkTimeLabels">
     <property name="text">
      <string>Show Time Labels on Graph</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkAlignRight">
     <property name="text">
      <string>Align Graph Labels Right</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Graph History Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinHistorySize">
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Graph Max Speed (MB/s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinMaxSpeed">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
